README.md

```diff
@@ -58,6 +58,7 @@ try{
  REST | js | Description  
 ----------------------------|---|-------------
 POST&nbsp;/v1/messages | `messages.send(message)` | Send a message to a contact. There are different message types for every platform ([FB](https://docs.amio.io/v1.0/reference#facebook-messenger-messages-text), [Viber](https://docs.amio.io/v1.0/reference#viber-messages-text)).
+POST&nbsp;/v1/notifications | `notifications.send(notification)` | Send a notification to a contact. There are notifications for [FB](https://docs.amio.io/v1.0/reference#facebook-messenger-notifications-send-notification) only.
```

Comment:
I would not mention for which platforms it is eligible. User can see it in docs. I would rather lead them to general notification link: https://docs.amio.io/v1.0/reference#notifications-send-notification